tests/integration/states/test_esxi.py

```diff
@@ -279,71 +279,71 @@ def test_vmkernel_adapter_present(vmware_datacenter, service_instance):
     assert not delete_ret["changes"]
 
 
-def test_maintenance_mode(service_instance):
-    hosts = list(esxi_mod.get(service_instance=service_instance))
+def test_maintenance_mode(patch_salt_globals):
```

Comment:
If this is an integration test, why isn't `service_instance` needed?

I don't see how `patch_salt_globals` is patching `utils.esxi.get_service_instance()` to return a service instance, or how `patch_salt_globals` is patching `utils.vmware._get_service_instance()`, either...

Reply:
It is passing the creds through the pillar in the patch globals.